2019/06/09 07:59:41 [emerg] 1912#18396: "proxy_pass" cannot have URI part in location given by regular expression, or inside named location, or inside "if" statement, or inside "limit_except" block in D:\Program Files\nginx/conf/nginx.conf:60
2019/06/09 08:00:53 [emerg] 1556#16260: "proxy_pass" cannot have URI part in location given by regular expression, or inside named location, or inside "if" statement, or inside "limit_except" block in D:\Program Files\nginx/conf/nginx.conf:60
2019/06/09 08:01:59 [emerg] 6988#22784: "proxy_pass" cannot have URI part in location given by regular expression, or inside named location, or inside "if" statement, or inside "limit_except" block in D:\Program Files\nginx/conf/nginx.conf:60
2019/06/09 08:03:31 [notice] 10356#19252: signal process started
2019/06/09 08:03:31 [error] 10356#19252: CreateFile() "D:\Program Files\nginx/logs/nginx.pid" failed (2: The system cannot find the file specified)
2019/06/09 08:04:25 [notice] 8972#16356: signal process started
2019/06/09 08:04:25 [error] 8972#16356: CreateFile() "D:\Program Files\nginx/logs/nginx.pid" failed (2: The system cannot find the file specified)
2019/06/09 08:07:22 [error] 4752#19372: *4 CreateFile() "E:/JSProject/oldking/dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1:9999", referrer: "http://127.0.0.1:9999/"
2019/06/09 08:14:15 [notice] 4276#14708: signal process started
2019/06/09 08:24:18 [notice] 12100#16968: signal process started
2019/06/09 08:24:51 [notice] 18924#22040: signal process started
2019/09/29 21:48:16 [error] 71292#9900: *4 connect() failed (10060: A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond) while connecting to upstream, client: 127.0.0.1, server: localhost, request: "GET /api/identifycode/pic HTTP/1.1", upstream: "http://167.86.122.142:981/identifycode/pic", host: "127.0.0.1:9999", referrer: "http://127.0.0.1:9999/"
2019/09/29 21:48:16 [error] 71292#9900: *4 CreateFile() "E:/JSProject/oldking/dist/favicon.ico" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "127.0.0.1:9999", referrer: "http://127.0.0.1:9999/"
2019/09/29 21:55:40 [error] 49160#47208: *3 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/login" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /login HTTP/1.1", host: "localhost:2081"
2019/09/29 21:56:04 [error] 49160#47208: *3 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/upload" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /upload HTTP/1.1", host: "localhost:2081"
2019/09/29 22:01:00 [error] 49160#47208: *9 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/upload" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /upload HTTP/1.1", host: "localhost:2081"
2019/09/29 22:01:13 [error] 49160#47208: *10 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/login" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /login HTTP/1.1", host: "localhost:2081"
2019/09/29 22:06:09 [error] 49160#47208: *15 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/login" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /login HTTP/1.1", host: "localhost:2081"
2019/09/29 22:06:29 [error] 49160#47208: *15 CreateFile() "G:/AllUsersData/Oldking/node/vuetifyhelloworld/dist/login" failed (2: The system cannot find the file specified), client: 127.0.0.1, server: localhost, request: "GET /login HTTP/1.1", host: "localhost:2081"
2019/09/29 22:10:34 [error] 49160#47208: *26 client intended to send too large body: 877445318 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:12:58 [error] 49160#47208: *31 client intended to send too large body: 877445318 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:15:28 [error] 49160#47208: *37 client intended to send too large body: 877445318 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:16:49 [error] 49160#47208: *43 client intended to send too large body: 465219790 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:18:07 [error] 49160#47208: *46 client intended to send too large body: 61795831 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:22:35 [notice] 35504#35332: signal process started
2019/09/29 22:22:53 [error] 49160#47208: *50 client intended to send too large body: 61795831 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 22:24:05 [error] 49160#47208: *55 client intended to send too large body: 7377068 bytes, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", host: "localhost:2081", referrer: "http://localhost:2081/"
2019/09/29 23:16:35 [error] 32960#25316: *20 WSASend() failed (10054: An existing connection was forcibly closed by the remote host) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/29 23:19:10 [error] 32960#25316: *29 WSASend() failed (10053: An established connection was aborted by the software in your host machine) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/29 23:23:02 [error] 32960#25316: *32 WSASend() failed (10053: An established connection was aborted by the software in your host machine) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/29 23:42:57 [emerg] 53744#52024: "root" directive is not allowed here in C:\Users\yiche\go\src\PeDataCenterServer\nginx/conf/nginx.conf:1
2019/09/29 23:54:15 [error] 15132#15284: *6 WSASend() failed (10054: An existing connection was forcibly closed by the remote host) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/30 00:01:32 [error] 9540#10044: *5 WSASend() failed (10054: An existing connection was forcibly closed by the remote host) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/30 02:25:07 [error] 5220#11300: *18 WSASend() failed (10053: An established connection was aborted by the software in your host machine) while sending request to upstream, client: 127.0.0.1, server: localhost, request: "POST /rpc/upload2tempfiles HTTP/1.1", upstream: "http://127.0.0.1:2083/upload2tempfiles", host: "127.0.0.1:2081", referrer: "http://127.0.0.1:2081/"
2019/09/30 02:34:09 [emerg] 3424#10464: bind() to 0.0.0.0:2081 failed (10013: An attempt was made to access a socket in a way forbidden by its access permissions)
